UPGRADING.md

```diff
@@ -1,5 +1,32 @@
 # Upgrading Replicated Security
 
+## Unreleased
+
+### Provider
+
+Upgrading a provider from v5.1.x requires state migrations. The following migrators should be added to the upgrade handler of the provider chain:
+
+```go
+// InitializeMaxValidatorsForExistingConsumers initializes the max validators
+// parameter for existing consumers to the MaxProviderConsensusValidators parameter.
+// This is necessary to avoid those consumer chains having an excessive amount of validators.
+func InitializeMaxValidatorsForExistingConsumers(ctx sdk.Context, providerKeeper providerkeeper.Keeper) {
```

Comment:
Shouldn't those functions appear in our migration code somewhere 🤔 ?

Reply:
Negative, see here for a discussion I had with Marius about this:
https://github.com/cosmos/interchain-security/pull/2079#discussion_r1694853611